### DISCUSSION TITLE
Using the bare esp32 chip

### DISCUSSION THREAD

**Discussion by valentinvirban** [alexanderlavrushko/BLE-HUD-navigation-ESP32#19] [Q&A] [answered]
Hi! Silly question! Since you only need the IO pins for the display, is it possible to unsolder the esp32 chip from the board after programing and use it for the project? I am new to arduino and got excited after finding your project and now i am searching for an old iphone to buy only for this app.thank you!

**Comment by alexanderlavrushko** [comment, marked as answer]
I don't think unsoldering is a good idea, all those components around ESP32 chip are definitely necessary for its proper functioning. The best option is to find a board which fits your needs.

Actually there is a chance that Android app will also support this in first half 2022.

**Reply by valentinvirban** [reply]
If this is the case, i can't wait for the android version! Thanks for answering!
